relay/nakama/main.go

```diff
@@ -167,8 +167,13 @@ func initEventHub(
 		ch := eventHub.SubscribeToEvents("main")
 		for event := range ch {
 			content := make(map[string]any)
-			err = json.Unmarshal(event, &content)
-			err := eris.Wrap(nk.NotificationSendAll(ctx, "event", content, 1, false), "")
+			err := json.Unmarshal(event, &content)
+			if err != nil {
+				// The event content isn't in JSON format. Wrap whatever it is in a JSON blob.
+				content["message"] = string(event)
+			}
+
+			err = eris.Wrap(nk.NotificationSendAll(ctx, "event", content, 1, false), "")
```

Comment:
not sure if this is meant to be a patch fix or not, but what if we just didn't unmarshal at all? IMO we should just be transferring the bytes received. not sure we gain anything by adding the generic unmarshal step

Reply:
like we could just do `content["message"]=event`?

Reply:
the `event` here should just be JSON marshalled `TickResults`, so i'm not actually sure whats causing the failure tbh

Reply:
nvm talked with jeremy, this might be ugly